concepticondata/references/references.bib

```diff
@@ -3102,11 +3102,22 @@ @Article{Yao2017
   Number                              = {4}
 }
 
-@book{Hartmann2013,
+@Book{Grierson1928,
+  author     = {George Abraham Grierson},
+  booktitle  = {Linguistic Survey of India},
+  editor     = {Grierson, George Abraham},
+  pages      = {viii+337+2},
+  publisher  = {Office of the Superintendent of Goverment Printing},
+  series     = {Linguistic Survey of India},
+  title      = {Comparative Vocabulary},
+  volume     = {I(II)},
+  year       = {1928}
+}
+
+@Book{Hartmann2013,
     publisher = {Max Planck Institute for Evolutionary Anthropology},
     title = {Valency Patterns Leipzig},
     url = {http://valpal.info},
     address = {Leipzig},
     editor = {Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley},
     year = {2013}
```

Comment:
One bracket missing here to close the bibtex, this was the error message.

Reply:
Added the closing bracket.